### PR TITLE
+ add flutter query network stats for all apps, return list apps with…

### PR DESCRIPTION
**Pull request by JacobKant**
+ add flutter query network stats for all apps, return list apps with  In and Out bytes transfered with any types of network by period from arguments

+ add coroutines for not freeze main thread

Related: https://github.com/Parassharmaa/usage_stats/issues/4

### CODE REVIEW

File: android/src/main/kotlin/io/github/parassharmaa/usage_stats/NetworkStats.kt [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    @RequiresApi(Build.VERSION_CODES.M)
+    private fun ApplicationInfo.getNetworkSummary(
+        networkType: Int,
+        networkStatsManager: NetworkStatsManager,
+        startDate: Long,
+        endDate: Long
+    ): AppNetworkStats {
+        try {
+            val queryDetailsForUid: NetworkStats = networkStatsManager.queryDetailsForUid(
+                networkType, "", startDate, endDate, uid
```
Comment (Parassharmaa) [inline android/src/main/kotlin/io/github/parassharmaa/usage_stats/NetworkStats.kt:76]: `uid`? I doubt if this works...

Comment (Parassharmaa) [reply android/src/main/kotlin/io/github/parassharmaa/usage_stats/NetworkStats.kt:76]: I have not yet tested the example code, just wanted to be sure about it once, before we merge.

Comment (JacobKant) [reply android/src/main/kotlin/io/github/parassharmaa/usage_stats/NetworkStats.kt:76]: Yes, you can check, I added the use of this API to the test example and it worked

queryDetailsForUid - is the only method that returns statistics for a specific application, it's strange but I always returned one bucket with statistics, although according to the documentation there should be several at time intervals, so I didn't make the history API.
made from this method while summary is in the context of applications

Comment (Parassharmaa) [reply android/src/main/kotlin/io/github/parassharmaa/usage_stats/NetworkStats.kt:76]: Yes, it works as described. Thanks!

**Comment by JacobKant** [comment]
@Parassharmaa sorry to interrupt, but could you put a label "hacktoberfest-accepted" on this pull request? )
this is new rules of hacktoberfest - https://hacktoberfest.digitalocean.com/hacktoberfest-update

**Comment by Parassharmaa** [comment]
Sure!

**Comment by mag1c-dev** [comment]
> * add flutter query network stats for all apps, return list apps with  In and Out bytes transfered with any types of network by period from arguments
> * add coroutines for not freeze main thread
> 
> Related: #4

Hello! I have an issue https://github.com/Parassharmaa/usage_stats/issues/27 with network stats. Please help me.

**Pull request merged**